kolibri/core/auth/serializers.py

```diff
@@ -40,20 +40,23 @@ class Meta:
 
     def create(self, validated_data):
         if FacilityUser.objects.filter(username__iexact=validated_data['username']).exists():
-            raise serializers.ValidationError(_('An account with that username already exists'))
+            raise serializers.ValidationError(detail={'username': ['An account with that username already exists.']},
+                                              code=error_constants.USERNAME_ALREADY_EXISTS)
```

Comment:
Since the user-facing string is no longer being translated here, I'm assuming this one is just for debugging, and it's now specified (and translated) in the frontend?